src/Patches/PlayerCharacterPatches.cs

```diff
@@ -203,19 +208,57 @@ public class PlayerCharacterPatches {
                             }
                         }
                     }
-                    InitialRewards.Add(Item.Reward);
+                    if (ProgressionNames.Contains(Item.Reward.Name) || RandomItemPatches.FairyLookup.Keys.Contains(Item.Reward.Name)) {
+                        ProgressionRewards.Add(Item.Reward);
+                    } else {
+                       InitialRewards.Add(Item.Reward);
+                    }
                     InitialLocations.Add(Item.Location);
                 }
             }
 
             // Randomize the rewards and locations
+            // put progression items in locations
```

Comment:
Could possibly move this block (221-261) to a "RandomizeItems" function to reduce clutter in this function

Reply:
Moved all of the item randomization code into its own function, since nothing else seemed to reference the lists that were created for it